MarketDataMessage.capnp

```diff
@@ -52,13 +52,13 @@ struct MarketDataRequest { # http://www.fixwiki.org/fixwiki/MarketDataRequest/FI
 #######################################################################################################
 
 struct MarketDataSnapshot { # http://fixwiki.org/fixwiki/MarketDataSnapshotFullRefresh/FIX.5.0SP2%2Bol
-    timestamp @0 :Float64;                  # required
```

Comment:
[Inquiry]
Is this message still in use? I am planning to refactor it from a scratch. If someone is using it, I need to create another file.

Reply:
@vitaliybezz Which message? Snapshot?

Reply:
I mean all MarketDataMessage.capnp

Reply:
Yes this is being used right now

Reply:
ok